script/Ciel/Day 5/05_01_CIEL05_1E.txt

```diff
@@ -26,7 +26,7 @@ The ceiling is supported by numerous pillars.
 -- 天井中心部には窓らしき穴があり、灰色の陽差しを下ろしている。それこそ、雲の隙間からのぞく<天使の梯子|ジェイコブズラダー>のように。
 // Getting ~biblical~ are we
 Grey sunlight leaks in through a window-like hole in the center of the ceiling.
- It almost like a Jacob's Ladder peeking through the gaps in the clouds.
+ It's almost like Jacob's Ladder peeking through the gaps in the clouds.
```

Comment:
Note so sure about this. "Jacob's ladder", as in the ladder to heaven, isn't something you'd refer to as "peeking from between the clouds". I'd imagine this is referring to the light phenomenon which is why the old version was phrased the way it is.
![image](https://user-images.githubusercontent.com/16092757/186270811-1a8572fa-596d-4ad3-bc56-99d4f846257e.png)

Reply:
to revert

Reply:
but "It's" is correct right?